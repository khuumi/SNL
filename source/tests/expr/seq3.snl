[58.1, "cow"]