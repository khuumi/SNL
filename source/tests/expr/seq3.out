List [(Float 58.1), (String cow)]